gallery/qip/plot_qip_amplitude_noise.py

```diff
@@ -0,0 +1,26 @@
+"""
+Control Amplitude noise
+=======================
+
+This example demonstrates how to add Gaussian noise to the control pulse.
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+from qutip.qip.device import Processor
+from qutip.qip import RandomNoise
+from qutip.operators import sigmaz, sigmay
+
+# add control Hamiltonians
+processor = Processor(N=1)
+processor.add_ctrl(sigmaz(), targets=0)
+processor.add_ctrl(sigmay(), targets=0)
+
+# define coeffs and tlist
+processor.coeffs = np.array([[ 0.3, 0.,  0.2],
```

Comment:
isn't the method `coeff`, and not `coeffs`?

Reply:
The method is actually `coeffs`, I hope I didn't make a mistake somewhere else calling it wrongly as `coeff`?

Reply:
I called it `coeffs` because it is a 2d-array where each row is the time-dependent coefficient for one Hamiltonian term